snippets/markdown.json

```diff
@@ -376,5 +376,19 @@
       "${0}"
     ],
     "description": "Insert table with 5 rows and 5 columns. First row is heading."
+  },
+  "Insert subscript": {
+    "prefix": "sub",
+    "body": [
+      "${1}<sub>${0}"
+    ],
+    "description": "Create a subscript."
+  },
+  "Insert superscirpt": {
```

Comment:
nit: typo on 'superscript'

Reply:
I noticed this after approving the first time, will wait a bit to merge so you can fix it

Reply:
Fixed the typo. :)